jubatus/core/nearest_neighbor/lsh_function.cpp

```diff
@@ -27,46 +27,64 @@
 
 using std::vector;
 using jubatus::core::storage::bit_vector;
+using jubatus::core::nearest_neighbor::cache_t;
+using jubatus::util::concurrent::scoped_lock;
 
 namespace jubatus {
 namespace core {
 namespace nearest_neighbor {
 namespace {
 
 std::vector<float> random_projection_internal(
-  const common::sfv_t& sfv, uint32_t hash_num, size_t start, size_t end);
+    const common::sfv_t& sfv, uint32_t hash_num,
```

Comment:
coding rule
https://gist.github.com/unnonouno/4655412#function-declarations-and-definitions